### ISSUE TITLE
Cronitor discover fatal error.

### ISSUE THREAD

**Issue by stevepsharpe** [cronitorio/cronitor-cli#10]
Ubuntu 21.04 (GNU/Linux 5.11.0-37-generic x86_64)
CronitorCLI version 28.4

I installed Cronitor using the following `curl https://cronitor.io/install-linux?sudo=1 -H "API-KEY: ABC123"  | sh` and when I run `cronitor discover` I get the following error.

Any help would be awesome. Thanks!

```fatal error: unexpected signal during runtime execution
[signal SIGSEGV: segmentation violation code=0x1 addr=0xe5 pc=0x7f3e1794dc74]

runtime stack:
runtime.throw({0x8f412f, 0x0})
	/usr/local/go/src/runtime/panic.go:1198 +0x71
runtime.sigpanic()
	/usr/local/go/src/runtime/signal_unix.go:719 +0x396

goroutine 1 [syscall]:
runtime.cgocall(0x7909b0, 0xc0000f5878)
	/usr/local/go/src/runtime/cgocall.go:156 +0x5c fp=0xc0000f5850 sp=0xc0000f5818 pc=0x4040bc
os/user._Cfunc_mygetpwuid_r(0x3e8, 0xc000268390, 0x2694d80, 0x400, 0xc000010148)
	_cgo_gotypes.go:172 +0x4c fp=0xc0000f5878 sp=0xc0000f5850 pc=0x7570ec
os/user.lookupUnixUid.func1.1(0x3e8, 0xe90680, 0xc0000f5910, 0x40dce7)
	/usr/local/go/src/os/user/cgo_lookup_unix.go:105 +0xa6 fp=0xc0000f58d8 sp=0xc0000f5878 pc=0x757826
os/user.lookupUnixUid.func1()
	/usr/local/go/src/os/user/cgo_lookup_unix.go:105 +0x29 fp=0xc0000f5908 sp=0xc0000f58d8 pc=0x757769
os/user.retryWithBuffer(0xc000237d10, 0xc0000f59e8)
	/usr/local/go/src/os/user/cgo_lookup_unix.go:249 +0x39 fp=0xc0000f5950 sp=0xc0000f5908 pc=0x757e59
os/user.lookupUnixUid(0x3e8)
	/usr/local/go/src/os/user/cgo_lookup_unix.go:98 +0x118 fp=0xc0000f5a28 sp=0xc0000f5950 pc=0x757578
os/user.current()
	/usr/local/go/src/os/user/cgo_lookup_unix.go:51 +0x47 fp=0xc0000f5a68 sp=0xc0000f5a28 pc=0x757447
os/user.Current.func1()
	/usr/local/go/src/os/user/lookup.go:15 +0x17 fp=0xc0000f5a78 sp=0xc0000f5a68 pc=0x757f37
sync.(*Once).doSlow(0xc0000f5b60, 0x40d994)
	/usr/local/go/src/sync/once.go:68 +0xd2 fp=0xc0000f5ae0 sp=0xc0000f5a78 pc=0x46c9d2
sync.(*Once).Do(...)
	/usr/local/go/src/sync/once.go:59
os/user.Current()
	/usr/local/go/src/os/user/lookup.go:15 +0x37 fp=0xc0000f5b10 sp=0xc0000f5ae0 pc=0x756ef7
github.com/cronitorio/cronitor-cli/cmd.glob..func5(0xe369a0, {0xec68f0, 0x0, 0x0})
	/github/workspace/cmd/discover.go:134 +0x3e fp=0xc0000f5d00 sp=0xc0000f5b10 pc=0x78d5be
github.com/spf13/cobra.(*Command).execute(0xe369a0, {0xec68f0, 0x0, 0x0})
	/go/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:844 +0x5f8 fp=0xc0000f5dc0 sp=0xc0000f5d00 pc=0x58def8
github.com/spf13/cobra.(*Command).ExecuteC(0xe361c0)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:945 +0x38d fp=0xc0000f5e78 sp=0xc0000f5dc0 pc=0x58e54d
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:885
github.com/cronitorio/cronitor-cli/cmd.Execute()
	/github/workspace/cmd/root.go:53 +0x25 fp=0xc0000f5ea0 sp=0xc0000f5e78 pc=0x7895c5
github.com/getsentry/raven-go.(*Client).CapturePanicAndWait(0x200000003, 0xc0000001a0, 0xc0000f5f70, {0x0, 0x404111, 0x790db0})
	/go/pkg/mod/github.com/getsentry/raven-go@v0.2.0/client.go:822 +0xac fp=0xc0000f5f20 sp=0xc0000f5ea0 pc=0x75342c
github.com/getsentry/raven-go.CapturePanicAndWait(...)
	/go/pkg/mod/github.com/getsentry/raven-go@v0.2.0/client.go:828
main.main()
	/github/workspace/main.go:41 +0x1c5 fp=0xc0000f5f80 sp=0xc0000f5f20 pc=0x790805
runtime.main()
	/usr/local/go/src/runtime/proc.go:255 +0x227 fp=0xc0000f5fe0 sp=0xc0000f5f80 pc=0x436767
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1581 +0x1 fp=0xc0000f5fe8 sp=0xc0000f5fe0 pc=0x4641e1```

**Comment by shaneharter** [comment]
Hi Steve, I have just published a new build, 28.5, that I believe will address this. 

The very technical cause for the mess-up is that when building I was unknowingly trying to statically link CGO which does not like to be statically linked.

**Comment by stevepsharpe** [comment]
Hey Shaun, thank you so much for the super quick fix. I'm up and running now! Truly appreciated.
